From 051757e83fd6d28ca1193646259295d686554d96 Mon Sep 17 00:00:00 2001
From: Kalyan Kondapally <kalyan.kondapally@intel.com>
Date: Sun, 30 Aug 2015 16:43:43 -0700
Subject: [PATCH 12/14] Use InputMethodAuraLinux for Ozone implementations on
 Linux

InputMethodAuraLinux provides an ui::InputMethod implementation
for Aura on Linux platforms. Currently, this is initialized only
when USE_X11 is true. We should also be able to use this for Ozone
implementations on non-chromeos platforms.
---
 ui/base/ime/input_method_factory.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ui/base/ime/input_method_factory.cc b/ui/base/ime/input_method_factory.cc
index 12e38ea..f2b39b0 100644
--- a/ui/base/ime/input_method_factory.cc
+++ b/ui/base/ime/input_method_factory.cc
@@ -14,7 +14,7 @@
 #include "ui/base/ime/remote_input_method_win.h"
 #elif defined(OS_MACOSX)
 #include "ui/base/ime/input_method_mac.h"
-#elif defined(USE_AURA) && defined(OS_LINUX) && defined(USE_X11) && \
+#elif defined(USE_AURA) && defined(OS_LINUX) && \
       !defined(OS_CHROMEOS)
 #include "ui/base/ime/input_method_auralinux.h"
 #elif defined(OS_ANDROID)
@@ -58,7 +58,7 @@ scoped_ptr<InputMethod> CreateInputMethod(
   return make_scoped_ptr(new InputMethodWin(delegate, widget));
 #elif defined(OS_MACOSX)
   return make_scoped_ptr(new InputMethodMac(delegate));
-#elif defined(USE_AURA) && defined(OS_LINUX) && defined(USE_X11) && \
+#elif defined(USE_AURA) && defined(OS_LINUX) && \
       !defined(OS_CHROMEOS)
   return make_scoped_ptr(new InputMethodAuraLinux(delegate));
 #elif defined(OS_ANDROID)
-- 
1.9.1

